stat/distuv/pareto.go

```diff
@@ -88,6 +88,14 @@ func (p Pareto) Prob(x float64) float64 {
 	return math.Exp(p.LogProb(x))
 }
 
+// Quantile returns the inverse of the cumulative probability distribution.
+func (p Pareto) Quantile(prob float64) float64 {
+	if prob < 0 || prob > 1 {
```

Comment:
The convention here is `prob < 0 || 1 < prob`. This makes the condition reflect the arrangement on the number line.

Reply:
OK. I copied this from another file ;-)

Reply:
Yeah, no worries. The conventions evolve.